### DISCUSSION TITLE
How to set cookie which is come from the server to browser

### DISCUSSION THREAD

**Discussion by shahensaifullah** [nuxt/nuxt#16102] [Questions]
A CSRF token is come from the server but i cannot set that token to cookie where my server response as 'set-cookie'. 

```
# plugins/token.client.js

import {useCookie, useFetch} from "nuxt/app";

export default defineNuxtPlugin(async nuxtApp => {
    const token = useCookie('csrftoken').value
    if (token === undefined) {
        await useFetch('http://127.0.0.1:8000/api/auth/check/csrf/')
    }
})
```
![image](https://user-images.githubusercontent.com/44544213/208637496-d6b2b133-92b2-4bec-b5b1-7c95e30ed2c8.png)
This is response but not set to browser.

How can i solve this problem????

**Comment by zernonia** [comment]
Have you check out [Handling Cookies in API Routes](https://nuxt.com/docs/api/composables/use-cookie#handling-cookies-in-api-routes)? 

You need to set that cookie from the server side.
